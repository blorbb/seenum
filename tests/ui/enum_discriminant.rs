use seenum::EnumSelect;

#[derive(EnumSelect)]
#[repr(usize)]
pub enum Something {
    Abc = 0,
    Xyz = 2,
}

fn main() {}
