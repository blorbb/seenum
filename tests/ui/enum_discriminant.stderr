error: all variants must have the default discriminant
 --> tests/ui/enum_discriminant.rs:6:11
  |
6 |     Abc = 0,
  |           ^
